Update README (#1027)
kubeshop/tracetest: README.md

# Tracetest

Tracetest is a trace-based testing tool that leverages the data captured by your exiting Open Telemetry distributed traces to produce easy to create, yet super powerful integration tests. You can verify activity deep inside your system by asserting on data and flow information contained in the OpenTelemetry traces and span attributes. This can include:

> - verify the quality of your OpenTelemetry instrumentation and enforce standards.

Reviewer: ```suggestion
- Verifying the quality of your OpenTelemetry instrumentation and enforcing standards.
```

> - testing events that occur on 'the other side' of an async message queue, even though the original async call has returned earlier.

Reviewer: ```suggestion
- Testing events that occur on 'the other side' of an async message queue, even though the original async call has returned earlier.
```

> - assertions based on the timing of different steps in your process.

Reviewer: ```suggestion
- Assertions based on the timing of different steps in your process.
```

> - wildcard assertions across common types of activities, ie all gRPC return codes should be 0, all database calls should happen in less than 100ms.

Reviewer: ```suggestion
- Wildcard assertions across common types of activities, ie all gRPC return codes should be 0, all database calls should happen in less than 100ms.
```

> - test long running processes instrumented with OpenTelemetry tracing to assert proper operation deep in the process.

Reviewer: ```suggestion
- Testing long running processes instrumented with OpenTelemetry tracing to assert proper operation deep in the process.
```

# Features

> - Supports multiple ways of creating a test, including HTTP, GRPC and Postman Collections.

Reviewer: ```suggestion
- Supporting multiple ways of creating a test, including HTTP, GRPC and Postman Collections.
```

> - [Add assertions](https://kubeshop.github.io/tracetest/adding-assertions/) based on return data from trigger call and/or data contained in the spans in your distributed trace.

Reviewer: ```suggestion
- [Adding assertions](https://kubeshop.github.io/tracetest/adding-assertions/) based on return data from trigger call and/or data contained in the spans in your distributed trace.
```

> - Specify which spans to check in assertions via the [advanced selector language](https://kubeshop.github.io/tracetest/advanced-selectors/).

Reviewer: ```suggestion
- Specifying which spans to check in assertions via the [advanced selector language](https://kubeshop.github.io/tracetest/advanced-selectors/).
```

> - Define checks against the attributes in these spans, including properties, return status, or timing.

Reviewer: ```suggestion
- Defining checks against the attributes in these spans, including properties, return status, or timing.
```

# Feedback

> Have an idea to improve Tracetest? Please [Create an issue here](https://github.com/kubeshop/tracetest/issues/new/choose) or join our community on [Discord](https://discord.gg/eBvEQRVyKX).

Reviewer: ```suggestion
Have an idea to improve Tracetest? Please [create an issue here](https://github.com/kubeshop/tracetest/issues/new/choose) or join our community on [Discord](https://discord.gg/eBvEQRVyKX).
```

> Follow us on [Twitter at @tracetest_io](https://twitter.com/tracetest_io) for updates

Reviewer: ```suggestion
Follow us on [Twitter at @tracetest_io](https://twitter.com/tracetest_io) for updates.
```
